frontend/js/src/user/layout.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from "react";
+import { NavLink, Outlet, useLocation } from "react-router-dom";
+import GlobalAppContext from "../utils/GlobalAppContext";
+
+function NavItem({
+  label,
+  url,
+  isActive,
+  isDisabled,
+}: {
+  label: string;
+  url: string;
+  isActive: boolean;
+  isDisabled?: boolean;
+}) {
+  return (
+    <li
+      className={`${isActive ? "active" : ""} ${isDisabled ? "disabled" : ""}`}
+    >
+      <NavLink to={url}>{label}</NavLink>
+    </li>
+  );
+}
+
+function UserFeedLayout() {
+  const location = useLocation();
+  const locationArr = location?.pathname?.split("/");
+  const { currentUser } = React.useContext(GlobalAppContext);
+  const sitewide = locationArr[1] !== "user";
+  const userName = sitewide ? currentUser?.name : locationArr[2];
+
+  const [activeSection, setActiveSection] = React.useState<string>(
+    sitewide ? locationArr[2] : locationArr[3]
+  );
+
+  React.useEffect(() => {
+    setActiveSection(locationArr[3]);
+  }, [locationArr]);
+
+  return (
+    <>
+      <div className="secondary-nav dragscroll nochilddrag">
+        <ul className="nav nav-tabs" role="tablist">
+          <li className="username">
+            {userName ? (
+              <div>{userName}</div>
```

Comment:
Just a quick request from aerozol for this to link to the user dashboard (essentially the same as the "Listens" navitem).
I was going to do it in the current html templates but it will be lost.